lib/hammer_cli_katello/repository.rb

```diff
@@ -255,6 +262,27 @@ def import_upload_ids(ids)
         }
         resource.call(:import_uploads, params)
       end
+
+      def progress_message(filename)
+        _('Uploading %{filename}') % {:filename => filename}
+      end
+
+      def render_result
+        print_message _('finished')
```

Comment:
Not sure I want to expand this pull-request into more than what's here. Perhaps this was not the "low hanging fruit" issue I had hoped it was. :)